internal/usecase/player_locked_song_usecase_impl.go

```
@@ -110,5 +119,66 @@ func (u *playerLockedSongUsecase) Unlock(ctx context.Context, userID int, input
 		return nil
 	}
-	return u.lockedRepo.Delete(ctx, u.db, player.ID, *songID, input.IsUltima)
+	return u.tm.Transactional(ctx, func(tx repository.Executor) error {
+		if err := u.lockedRepo.Delete(ctx, tx, player.ID, *songID, input.IsUltima); err != nil {
+			return err
+		}
+		return u.recalculatePlayerOverpowerWithTx(ctx, tx, player)
+	})
+}
+
+func (u *playerLockedSongUsecase) recalculatePlayerOverpower(ctx context.Context, player *entity.Player) error {
```

golangci-lint (2.12.1)

[error] func (*playerLockedSongUsecase).recalculatePlayerOverpower is unused

(unused)